android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetViewModel.kt

```diff
@@ -83,18 +85,22 @@ constructor(val defaultRepository: DefaultRepository, val dispatcherProvider: Di
     val familiesList = ArrayList<Pair<Group, Location>>()
 
     familiesWithLocations.forEach { family ->
-      val familyLocation =
-        defaultRepository.fhirEngine.get<Location>(
-          family.characteristic.firstOrNull { characteristic ->
-              characteristic.value is Reference &&
-                characteristic.valueReference.reference.contains("Location")
-            }!!
-            .valueReference
-            .referenceElement
-            .idPart
-        )
-
-      familiesList.add(Pair(family, familyLocation))
+      try {
+        val familyLocation =
+          defaultRepository.fhirEngine.get<Location>(
+            family.characteristic.firstOrNull { characteristic ->
+                characteristic.value is Reference &&
+                  characteristic.valueReference.reference.contains("Location")
+              }!!
+              .valueReference
+              .referenceElement
+              .idPart
+          )
+
+        familiesList.add(Pair(family, familyLocation))
+      } catch (ex: com.google.android.fhir.db.ResourceNotFoundException) {
```

Comment:
Add import for `ResourceNotFoundException`

Reply:
Fixed